src/less/textbox/ds6/textbox.less

```diff
@@ -0,0 +1,4 @@
+@import "../../less/ds6/variables.less";
+@import "textbox-mixin.less";
```

Comment:
This file should be like:

```
@import "../../less/ds6/variables.less";
@import "../../less/ds6/mixins.less";

@import "textbox-mixins.less";
@import "textbox-base.less";
```

Note the `textbox-mixins.less` (plural) is added. The mixins should not live in the `textbox-mixin.less` (singular) file. The singular-named file is _only_ for creating the scoped CSS.

Therefore, you should create a `textbox-mixins.less` file and put the mixins for this component in there.

Please reach out to me if you have questions.